packages/lumx-react/src/components/flag/Flag.test.tsx

```diff
@@ -0,0 +1,86 @@
+import { mount, shallow } from 'enzyme';
+import 'jest-enzyme';
+import React, { ReactElement } from 'react';
+
+import { ColorPalette, Theme } from '@lumx/react';
+import { mdiAbTesting } from '@lumx/icons';
+import { Wrapper } from '@lumx/react/testing/utils';
+import { getBasicClass } from '@lumx/react/utils';
+import { Flag, FlagProps, CLASSNAME } from './Flag';
+
+/**
+ * Mounts the component and returns common DOM elements / data needed in multiple tests further down.
+ */
+const setup = (propOverrides: Partial<FlagProps> = {}, shallowRendering = true) => {
+    const props = {
+        label: 'default',
+        ...propOverrides,
+    };
+    const renderer: (el: ReactElement) => Wrapper = shallowRendering ? shallow : mount;
+    const wrapper = renderer(<Flag {...props} />);
+
+    return {
+        props,
+        iconEl: wrapper.find('Icon'),
+        wrapper,
+    };
+};
+
+describe('<Flag />', () => {
+    // 1. Test render via snapshot (default state of component).
+    describe('Snapshot', () => {
```

Comment:
Could you use `itShouldRenderStories` instead? This method will generate snapshot based on your stories which cover more cases that your current test.

Also, could you use `commonTestsSuite`? This method tests that your component can forward props and className properly.

Here again, you can look at the Badge component for inspiration.